specification/search/data-plane/Microsoft.Azure.Search.Data/preview/2017-11-11-preview/searchindex.json

```diff
@@ -668,6 +668,15 @@
               "name": "AutocompleteParameters"
             }
           },
+          {
+            "name": "$filter",
+            "in": "query",
+            "type": "string",
+            "description": "An OData expression that filters the completed terms returned as part of Autocomplete result.",
```

Comment:
It isn't clear from the description *how* the filter filters completed terms, since the filter itself is expressed in terms of documents and fields, not terms. How about:

    An OData expression that filters the documents used to produce completed terms for the Autocomplete result.

Reply:
Sounds better.